src/templates/token.tsx

```diff
@@ -271,9 +373,9 @@ const generateTokenPage = ({ secret }: { secret: string }) => {
 }
 
 export const generateTokenHtml = (
-  { secret }: { secret: string },
+  { secret, domain }: { secret: string; domain: string },
```

Comment:
Let's use an interface for this and for `generateTokenPage` in `token.tsx`

Reply:
Added